Modix.Services/FileUpload/FileUploadBehavior.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using Modix.Services.Moderation;
+using Serilog;
+
+namespace Modix.Services.FileUpload
+{
+    public class FileUploadBehavior : BehaviorBase
```

Comment:
A more useful name here might be `FileUploadPurgingBehavior`? Or `AttachmentBlacklistBehavior`?

Also, does anyone else think it makes sense to put this in `Modix.Services.Moderation` as it only really uses `IModerationService` and because `InvitePurgingBehavior` is also there?